content/backmatter.tex

```diff
@@ -190,6 +190,24 @@ \chapter{Undefined Behavior in OpenSHMEM}\label{sec:undefined}
 with the \LibConstRef{SHMEM\_CTX\_PRIVATE} option enabled; otherwise, the
 behavior is undefined.\tabularnewline
 \hline
+Creating a team with a $stride$ value equal to 0 and the $size$ value not equal to 1 &
+If a $stride$ value equal to 0 is passed to \FUNC{shmem\_team\_split\_strided},
```

Comment:
Should we use \VAR for stride?

Reply:
Fixed -- thanks for catching this! I posted #558 to also fix the source of the formatting issue.

Reply:
Actually, it's correct to format these in math mode in the source text. Closing #558.